Meadowlark-js/packages/meadowlark-authz-server/src/security/TokenValidator.ts

```diff
@@ -107,12 +107,14 @@ export async function introspectBearerToken(bearerToken: string, traceId: string
 
     const nJwtErrorMessages = ['Signature verification failed', 'Jwt cannot be parsed'];
 
+    // Check for error response from nJwt
     if (jwt == null || nJwtErrorMessages.includes(jwt.message)) return { isValid: false };
 
-    const isNotExpired: boolean = jwt.message !== 'Jwt is expired';
-    const isMeadowlarkToken: boolean = jwt.body?.iss === TOKEN_ISSUER && jwt.body?.aud === TOKEN_ISSUER;
+    // Check for correct issuer
+    if (jwt.body?.iss !== TOKEN_ISSUER || jwt.body?.aud !== TOKEN_ISSUER) return { isValid: false };
 
-    const active: boolean = isNotExpired && isMeadowlarkToken && (await isValidClientId(jwt.body?.client_id, traceId));
+    const isNotExpired: boolean = jwt.message !== 'Jwt is expired';
```

Comment:
I wondered where this text comes from. Answer: [a configurable file](https://github.com/jwtk/njwt/blob/master/properties.json).

This might not be worthwhile... wondering if we can do this?

```
import nJwtProperties from 'njwt/properties.json';

... 

const isNotExpired: boolean = jwt.message !==nJwtProperties.errors.EXPIRED;
```

Reply:
I'll take a look 👍

Reply:
A good idea, but we can't reach in there to the `properties.json` without file system hacking. It doesn't look exported.